gravitee-apim-console-webui/src/management/api/component/health-check-v4-form/api-health-check-v4-form.component.scss

```
@@ -0,0 +1,34 @@
+.health-check {
+  &__enableField {
```

According to what you wrote in the template it should be

```suggestion
  &__enable-field {
```